tests/transports-integrations/tests/utils/common.py

```
@@ -151,6 +151,75 @@ class Config:
+    "degrees",
+    "hot",
+    "cold",
+    "warm",
+    "cool",
+]
+
+LOCATION_KEYWORDS = ["boston", "san francisco", "new york", "city", "location", "place"]
+
+# Error test data for invalid role testing
+INVALID_ROLE_MESSAGES = [
+    {"role": "tester", "content": "Hello! This should fail due to invalid role."}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)